### PR TITLE
Docs additions for v1.5.0

### LINKED ISSUE #1019: API and Usage Doc documentation needs to be added for Change Password Functionality
## Abstract

https://github.com/rotki/rotki/pull/978 was implemented to address https://github.com/rotki/rotki/issues/763 (allow user to change their password), however the API and Usage Guide were not updated.

## Spec

Update the relevant files and be sure to include clear instructions accompanied by screenshots in the Usage Guide.
https://github.com/rotki/rotki/blob/develop/docs/api.rst#handling-user-creation-sign-in-log-out-and-querying
https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst

### PR DESCRIPTION
**Pull request by LefterisJP**
Fix #1019

### CODE REVIEW

**Comment by isidorosp** [comment]
There's a few cases in `api.rst` where the watcher endpoint has a type and is `/watcher/` instead of `/watchers/` (I can't figure out how to comment/link to sections that aren't in the diff).

File: docs/usage_guide.rst [hunk 1]
```diff
@@ -116,7 +117,7 @@ Set the display format of the dates in the Rotki user interface.
 Customizing the accounting settings
 ====================================
 
-By clicking on the user icon on the top right and choosing accounting settings you can customize some the settings that are used when calculating profit/loss for all your actions.
+By choosing the "Accounting" settings button you can customize some application settings that pertain to accounting calculations.
 
 You should understand what each setting does, consult with a tax accountant for your jurisdcition and then set them appropriately.
```
Comment (isidorosp) [inline docs/usage_guide.rst:122]: jurisdcition => jurisdiction

File: docs/api.rst [hunk 2]
```diff
@@ -3461,7 +3499,7 @@ Adding new watcher
             }],
       }
 
-   :reqjson list[object] watchers: A list of watchers to add as defined in the `above section <watchers_shema_section_>`__ but without an identifier. The identifier is created server-side and returned in the response.
+   :reqjson list[object] watchers: A list of watchers to add as defined in the `above section <watchers_shema_section>`__ but without an identifier. The identifier is created server-side and returned in the response.
```
Comment (isidorosp) [inline docs/api.rst:3502]: shema => schema
also fixes the link

File: docs/api.rst [hunk 3]
```diff
@@ -3520,7 +3558,7 @@ Editing watchers
             }]
       }
 
-   :reqjson list[object] watchers: A list of watcher to edit. As defined `here <watchers_shema_section_>`__.
+   :reqjson list[object] watchers: A list of watcher to edit. As defined `here <watchers_shema_section>`__.
```
Comment (isidorosp) [inline docs/api.rst:3561]: shema => schema
also fixes the link

File: docs/usage_guide.rst [hunk 4]
```diff
@@ -339,6 +350,19 @@ Without a premium subscription you also can't see the amount of DAI earned per a
...
+
+In the borrowing section you can find information on your makerdao vaults. The vaults are autodetected from your ethereum accounts and information about them is shown.
+
+As a normal non-premium user you can see all your vaults, and for each one inspect the locked collateral, collateralization, debt generated and the liquidation price.
+
+With a premium subscription you can also see additional information such as the creation time of the vault, list of historical activities, total interest owed and liquidation events.
+
+Premium users can also have makervault interest taken into account in the profit/loss report.
+
+Finally premium users can create watchers for their vaults. They can add a target collateralization ratio they would like rotki to watch for in a vault. If the collateralization ration becomess less/greater than that ratio then an alert is sent to the user's email. This watcher service runs in the rotki server so you don't even need to leave the application open.
```
Comment (isidorosp) [inline docs/usage_guide.rst:364]: becomess => becomes

runs in => runs on

the user's => your email (since we use second person throughout)

**Comment by LefterisJP** [comment]
@isidorosp thanks for finding the errors. Fixed in the latest commit.

**Pull request merged**